spokestack/wakeword/tflite.py

```diff
@@ -178,6 +184,7 @@ def _detect(self, context: SpeechContext) -> None:
 
         if posterior > self._posterior_threshold:
             context.is_active = True
+            _LOG.info(f"wake: {self._posterior_max}")
```

Comment:
we want to log the maximum posterior after we update it (I believe this was also wrong in android), so let's move lines 188-189 before 185